netmap/node_info.go

```diff
@@ -178,6 +179,11 @@ func (x NodeInfo) PublicKey() []byte {
 	return x.m.GetPublicKey()
 }
 
+// StringifyPublicKey returns HEX representation of PublicKey
+func (x NodeInfo) StringifyPublicKey() string {
```

Comment:
Can we make it a function over `NodeInfo`? It provides access to public key via method.

Reply:
Not sure that I understand your point, please check new changes.

Reply:
I mean now you can declare it as func `StringifyNodeKey(node NodeInfo) string` since no private data is used.

Reply:
Got it, fixed.